src/sass/layout/_contacts.scss

```diff
@@ -0,0 +1,298 @@
+.contacts {
+  padding-bottom: 60px;
+
+  @include tablet {
+    padding-bottom: 40px;
+  }
+
+  @include desktop {
+    padding-bottom: 120px;
+  }
+}
+
+.contacts__list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 16px;
+  margin-bottom: 24px;
+
+  @media screen and (max-width: 479px) {
+    margin-bottom: 20px;
+  }
+
+  @include tablet {
+    margin-bottom: 40px;
+  }
+
+  @include desktop {
+    gap: 32px;
+    margin-bottom: 60px;
+  }
+}
+
+.contacts__item {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  max-width: 280px;
+  max-height: 412px;
+  padding: 40px 36px 24px 32px;
+  background-color: #ffffff;
+  box-shadow: 0px 8px 30px rgba(212, 20, 67, 0.1);
+  border-radius: 24px;
+
+  @include tablet {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    flex-basis: calc((100% - 32px) / 3);
+    max-width: 224px;
+    max-height: 537px;
+    padding: 40px 42px 40px 24px;
+  }
+
+  @include desktop {
+    flex-basis: calc((100% - 64px) / 3);
+    max-width: 368px;
+    max-height: 542px;
+    padding: 40px;
+  }
+}
+
+.contacts__pre-title {
+  display: flex;
+  width: max-content;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(14 / 12);
+  letter-spacing: 0.04em;
+  color: getColor('theme-black');
+  background-color: getColor('yellow-main');
+  border-radius: 6px;
```

Comment:
border-radius: 4px; - for mobile screen and 480px

Reply:
fixed